openmc/data/library.py

```diff
@@ -112,6 +114,18 @@ def from_xml(cls, path):
 
         data = cls()
 
+        # If path is None, get the cross sections from the
+        # OPENMC_CROSS_SECTIONS environment variable
+        if path is None:
+            path = os.environ.get('OPENMC_CROSS_SECTIONS')
+
+        # Check to make sure there was an environmental variable.
+        if path is None:
+            raise ValueError("Either path or OPENMC_CROSS_SECTIONS "
+                             "environmental variable must be set")
+
+        check_type('path', path, str)
```

Comment:
Please add an import: `from six import string_types` and then change str to string_types.